tox.ini

```diff
@@ -1,12 +1,12 @@
 [tox]
-envlist = py37
+envlist = py310
```

Comment:
should this actually test multiple versions? lib users might still be running on 3.8 before they are able to migrate.

Reply:
python 3.8 and below have been deprecated and reached EOL. So we will only support python3.9 and above.